### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by trinhan-nguyen**

### CODE REVIEW

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (ntn20112006) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:5]: Need newlines

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (ntn20112006) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml:5]: Need newlines

File: app/src/main/res/values/ic_launcher_background.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#2196F3</color>
+</resources>
```
Comment (ntn20112006) [inline app/src/main/res/values/ic_launcher_background.xml:4]: Need newlines

**Review by ntn20112006 — CHANGES_REQUESTED** [review]

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml (hunk 1 above)
Comment (trinhan-nguyen) [reply app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: Updated

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml (hunk 2 above)
Comment (trinhan-nguyen) [reply app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: Updated

File: app/src/main/res/values/ic_launcher_background.xml (hunk 3 above)
Comment (trinhan-nguyen) [reply app/src/main/res/values/ic_launcher_background.xml]: Updated

**Review by ntn20112006 — APPROVED** [review]

**Pull request merged**